spec/policy_chain_spec.lua

```diff
@@ -92,6 +92,38 @@ describe('policy_chain', function()
     end, 'readonly table')
   end)
 
+  describe('.insert', function()
+
+    it('adds policy to the chain', function()
```

Comment:
I'd modify this test or add a new one. For `.insert` we need to test that it adds the policy to the end of the list. If we do not initialize the chain with a few elements, we won't be able to test that. This test would pass even if the code added the policy to the beginning of the list or in a random position.

Reply:
Good point 👍 Will fix.

Reply:
Fixed in https://github.com/3scale/apicast/pull/496/commits/d340bf1b423fd68189419873e5e098607930abcd but going to be squashed.